lblod/frontend-form-builder
# SFB-123: SHACL shape validation v2 forms

## ID
 [SFB-123](https://binnenland.atlassian.net/browse/SFB-123)

 ## Description

Alot of ttl code is generated with the form-builder. This code is not validated yet. Validations are checked with custom methods if they contain all the correct properties to add the validation but that is it.

Create `nodeShapes` for all the items in the formTtl for `V2 forms`: 

* field
* section
* required constraint
* max characters constraint

 ## Type of change


 ## How to test

1. Create a new form
2. Update it with the fields that you like
3. Remove a predicate or something that should be there in the triples
4. See if it is validated correctly

 ## Links to other PR's

- /
